README.md

```diff
@@ -0,0 +1,33 @@
+# SLB
+
+> C++ Standard Library Backport
+
+[![travis][badge.travis]][travis]
+[![appveyor][badge.appveyor]][appveyor]
+[![license][badge.license]][license]
```

Comment:
Missing definitions for refs `travis`, `appveyor`, `license`.

Reply:
Good catch! Thanks.